tests/lib/config/config-file.js

```diff
@@ -497,6 +519,32 @@ describe("ConfigFile", function() {
             assert.deepEqual(config, updatedConfig.eslintConfig);
         });
 
+        it("should load fresh information from a .eslintrc.js file", function() {
+            var initialConfig = {
+                    parserOptions: {},
+                    env: {},
+                    globals: {},
+                    rules: {
+                        quotes: [2, "double"]
+                    }
+                },
+                updatedConfig = {
+                    parserOptions: {},
+                    env: {},
+                    globals: {},
+                    rules: {
+                        quotes: 0
+                    }
+                },
+                tmpFilename = ".eslintrc.js",
+                tmpFilePath = writeTempJsConfigFile(initialConfig, tmpFilename),
+                config = ConfigFile.load(tmpFilePath);
+            assert.deepEqual(config, initialConfig);
+            writeTempJsConfigFile(updatedConfig, tmpFilename, path.dirname(tmpFilePath));
```

Comment:
This would leave temp file hanging around after tests, right? You should probably clean it up.